lang/MoarVM/files/patch-syncsocket.diff

```diff
@@ -0,0 +1,13 @@
+--- src/io/syncsocket.c.orig	2022-04-24 22:49:11.000000000 +0800
```

Comment:
Optional, but I would put the link to https://github.com/MoarVM/MoarVM/issues/1704 here as well which makes it easier to find it than by hunting for old PR comments. Same for the other one.

Reply:
Done.